jaeger-core/src/test/java/com/uber/jaeger/TracerTest.java

```diff
@@ -136,6 +138,22 @@ public void testServiceNameNotEmptyNull() {
   }
 
   @Test
+  public void testBuilderIsServerRPC() {
+    Tracer.SpanBuilder spanBuilder = (Tracer.SpanBuilder) tracer.buildSpan("ndnd");
+    spanBuilder.withTag(Tags.SPAN_KIND.getKey(), "server");
+
+    assertTrue(spanBuilder.isRPCServer());
+  }
+
+  @Test
+  public void testBuilderIsNotServerRPC() {
+    Tracer.SpanBuilder spanBuilder = (Tracer.SpanBuilder) tracer.buildSpan("Lrrr");
+    spanBuilder.withTag(Tags.SPAN_KIND.getKey(), "peachy");
```

Comment:
well good to know some people's servers are peachy.. wait.. I guess they aren't servers if they are peachy? :)